deltachat-ios/Controller/QrPageController.swift

```diff
@@ -9,7 +9,7 @@ class QrPageController: UIPageViewController, ProgressAlertHandler {
 
     private var selectedIndex: Int = 0
 
-    private func getQrCodeHint() -> String {
+    private var getQrCodeHint: String {
```

Comment:
no blocker, this is apple-swift-style, i know. but _i_ find that somewhat confusing, when such a thing is _accessed_, you simply do not know, by just seeing the variable name, what actually happens.

therefore, i prefer just functions.

don't make me think.